main.go

```diff
@@ -63,12 +63,14 @@ func main() {
 	r.Get("/assets/{asset}", routes.Asset)
 	r.Get("/callback", routes.Callback)
 	r.Get("/callback/dev", routes.CallbackDev)
+	r.Get("/callback/discord", routes.CallbackDiscord)
 	r.Get("/feeds", routes.Feeds)
 	r.Get("/feeds/{feed}", routes.Feed)
 	r.Route("/golfer", func(r chi.Router) {
 		r.Use(middleware.GolferArea)
 		r.Post("/cancel-delete", routes.GolferCancelDelete)
 		r.Post("/delete", routes.GolferDelete)
+		r.Post("/disconnect-discord", routes.GolferDisconnectDiscord)
```

Comment:
How about /disconnect/discord, I can see us wanting to disconnect other connections in the future when we support multiple oauth providers.